pymongo/operations.py

```diff
@@ -593,23 +593,28 @@ def __init__(
         definition: Mapping[str, Any],
         name: Optional[str] = None,
         type: Optional[str] = "search",
```

Comment:
Do we want to remove this default?

Reply:
I'll ask that question in https://github.com/mongodb/specifications/pull/1541.